src/frontend/hooks/useSjekklikheter.ts

```diff
@@ -0,0 +1,30 @@
+import { useState, useCallback } from 'react';
+import { useHttp } from '@navikt/familie-http';
+import { type Ressurs, RessursStatus } from '@navikt/familie-typer';
+
+const useSjekkLikhetPerioder = (behandlingId: string) => {
+    const [result, setResult] = useState<Ressurs<boolean>>({ status: RessursStatus.IKKE_HENTET });
+    const { request } = useHttp();
+
+    const hentSjekkLikhetPerioder = useCallback(() => {
+        setResult({ status: RessursStatus.HENTER });
+
+        request<void, boolean>({
+            method: 'GET',
+            url: `/familie-tilbake/api/dokument/sjekk-likhet-perioder/${behandlingId}`,
```

Comment:
Burde denne likhetssjekken skje på "behandlingscontroller"-nivå istedenfor dokumentnivå?

Reply:
Vi lurte litt på det, eneste argumentet for å legge det i dokument, var at det skal ende med at vi slår sammen perioder i brevet hvis det er likt